File: Programming Language(ver.2)/Python/706.py
l = []
for i in range(4):
    l.append(int(input()))
with open('read.txt','r') as f:
    for i in f:
        l.append(int(i))
l.sort()
with open('write.txt','w') as f:
    for i in l:
        print(i)
        f.write(str(i)+'\n')

File: Programming Language(ver.2)/PHP/706.php
<?php
    $array = array();
    $file = fopen("read.txt", "r") or die("Unable to open the file");
    for($i = 0; $i < 4; $i++){
        array_push($array, intval(fgets(STDIN)));
    }
    while(!feof($file)){
        array_push($array, intval(fgets($file)));
    }
    sort($array);
    $result_file = fopen("write.txt", "w") or die("Unable to create the file");
    for($i = 0; $i < count($array); $i++){
        printf("%d\n", $array[$i]);
    }
    fwrite($result_file, sprintf("%s", join("\n", $array)));
?>